config/locales/en/nsm/steps.yml

```diff
@@ -152,6 +152,9 @@ en:
             duplicate: Duplicate
             delete: Delete
           unknown_date: Unknown date
+          items_imported:
+            one: You imported %{count} work item
```

Comment:
Does this also show correctly for 0?

Reply:
Shows like this: 
<img width="937" alt="image" src="https://github.com/user-attachments/assets/0086740d-e98e-4da9-99c4-f421078a368e" />
which is to spec, but not sure if it's something we want to show Mary to ask what to show or should it show at all of none have been imported